README.md

```diff
@@ -1,11 +1,17 @@
-## Description
-[Nest](https://github.com/nestjs/nest) framework TypeScript starter repository.
+# EddieBot
+
+The official [EddieBot](https://github.com/EddieHubCommunity/EddieBot.git) for the official [Discord server](http://discord.eddiehub.org). Join us at [Discord](http://discord.eddiehub.org) today!
+
+## Features
+
+- A powerful [API](https://github.com/EddieHubCommunity/api.git) built by the EddieHub community.
```

Comment:
Maybe we don't mention the API as a feature? (since it's separate tooling)

Reply:
Hmm well. I added it so the list looks more complete (I thought it was part of the bot). What else could we add? 🤔

Reply:
Does EddieBot uses this api? if yes then we can write like this. This is just a suggestion 😄 
```suggestion
- Uses a powerful [API](https://github.com/EddieHubCommunity/api.git) built by the EddieHub community.
```